CMakeLists.txt

```diff
@@ -1,10 +1,14 @@
 cmake_minimum_required (VERSION 3.2)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 project(sqlite_orm)
 
 option(SqliteOrm_BuildTests "Build sqlite_orm unit tests" ON)
 
 set(SqliteOrm_INCLUDE "${CMAKE_CURRENT_SOURCE_DIR}/include")
+add_library(sqlite_orm INTERFACE)
+target_include_directories(sqlite_orm INTERFACE include)
```

Comment:
```
set(SqliteOrm_INCLUDE "${CMAKE_CURRENT_SOURCE_DIR}/include")
```
is never used anymore, since `examples` and `tests` now just depend on `sqlite_orm`, which in turn declares its `include`-directory. However, I decided to keep this line since users of your library might depend on it. Not sure if you'd want to deprecate it though?